packages/cashier/src/Components/payment-agent-list.jsx

```diff
@@ -3,20 +3,21 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {
     Accordion,
-    Button,
     DesktopWrapper,
     Dropdown,
     Loading,
     MobileWrapper,
     SelectNative,
     Text,
+    Tabs,
 } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
-import { website_name } from '@deriv/shared';
+import { website_name, isDesktop } from '@deriv/shared';
```

Comment:
Could you consider alphabetic order when importing here and for importing from `@deriv/components`

Reply:
noted, done